persistent-sqlite/test/main.hs

```diff
@@ -216,6 +217,7 @@ main = do
     TransactionLevelTest.specsWith db
     MigrationTest.specsWith db
     LongIdentifierTest.specsWith db
+    PersistLiteralTestSQL.specsWith db
```

Comment:
I'm happy to accept the PR without fully supporting SQLite. We can mark this test as pending with `xdescribe`

```suggestion
    xdescribe "SQLite doesn't work, see issue #??? for discussion" $ do
        PersistLiteralTestSQL.specsWith db
```